recipes/tangled-up-in-unicode/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "tangled-up-in-unicode" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tangled_up_in_unicode-{{ version }}.tar.gz
```

Comment:
fyi, instead of `tangled_up_in_unicode` in the `url` you can use `{{ name | replace("-", "_") }}`.